packages/@okta/i18n/src/properties/login.properties

```diff
@@ -549,6 +549,22 @@ password.complexity.no_username = no parts of your username
 password.complexity.no_first_name = does not include your first name
 password.complexity.no_last_name = does not include your last name
 
+# Requirements as HTML list.
+# These are copy of existing strings, but we need them since in a list, the starting letter has to be in CAPS and no period at the end.
+password.complexity.requirements.html = Password requirements:
```

Comment:
Still not a huge fan of the ".html" suffixes especially since they don't actually contain any html.

How do you feel about `password.complexity.requirements.header` and `password.complexity.length.li` for the rest?

Reply:
Okay! Will change them.